### ISSUE TITLE
Documentation for the demo page

### ISSUE THREAD

**Issue by anevins12** [ryersondmp/sa11y#1]
Hey,

Demo page: https://ryersondmp.github.io/sa11y/

I was kindly notified about Sa11y from the web-a11y slack group with the explanation that it was designed with content editors in mind from CMSs. I understand how that works well with the workflow of content editors, but I couldn't quite get that from the documentation on the demo page. It would be great to have a section that explains the reason why it's great for CMS integration and how you envisage it would be used.

Thanks!

**Comment by ryersondmp** [comment]
Hi Andrew,

Thanks for the feedback! I admit I was in a bit of a rush to release the tool just in time for #GAAD. Although I'm definitely looking into expanding the documentation and providing a little video demo of how we've customized the tool to work with our content management system.

I'll give you an update soon!

Cheers,

Adam Chaboryk

**Comment by adamchaboryk** [comment]
Hi @anevins12,

Sorry for the super late reply. Although I hope the [new project website](https://sa11y.netlify.app/) and turnkey [WordPress plugin](https://sa11y.netlify.app/wordpress/) answers your questions!

Cheers
